cypress/integration/console/packet-broker/routing-policies.spec.js

```diff
@@ -87,6 +87,9 @@ describe('Packet Broker routing policies', () => {
   })
 
   it('succeeds setting individual per-network routing policy', () => {
+    cy.intercept('GET', '/api/v3/pba/home-networks/gateway-visibilities/default', {
+      statusCode: 404,
+    })
     cy.intercept('GET', '/api/v3/pba/home-networks/policies/default', {
```

Comment:
Since all tests in this suite use this intercept, we can just add it do `.beforeEach()` on LL20.